Scripts/Swift/Extensions/FileManager+Utils.swift

```diff
@@ -42,4 +42,31 @@ extension FileManager {
             print("Error \(error)")
         }
     }
+
+    func replaceAllOccurrences(of original: String, to replacing: String) throws {
+        let files = try allFiles(in: currentDirectoryPath)
+        for file in files {
+            do {
+                let text = try String(contentsOf: file, encoding: .utf8)
+                let modifiedText = text.replacingOccurrences(of: original, with: replacing)
+                try modifiedText.write(to: file, atomically: true, encoding: .utf8)
+            }
+            catch {}
+        }
+    }
+
+    private func allFiles(in directory: String) throws -> [URL] {
+        let url = URL(fileURLWithPath: directory)
+        var files = [URL]()
+        if let enumerator = enumerator(at: url, includingPropertiesForKeys: [.isRegularFileKey], options: [.skipsHiddenFiles, .skipsPackageDescendants]) {
```

Comment:
An improvement for indent

```suggestion
        if let enumerator = enumerator(
            at: url, 
            includingPropertiesForKeys: [.isRegularFileKey], 
            options: [.skipsHiddenFiles, .skipsPackageDescendants]
        ) {
```